### ISSUE TITLE
[Feature] Independent show Info, and grid view

### ISSUE THREAD

**Issue by FahimAnayet** [marta-file-manager/marta-issues#1020]
Hi, 
Thank you for this amazing file manager. I am using this file manager for over a year now. But I can not replace it with Finder because of some lacking feature. Although available features are enough for my use-case, and it's quite snappier than finder, but It depends on finder while showing file info. So I can not unload Finder (you can not kill, but can unload finder `launchctl unload /System/Library/LaunchAgents/com.apple.Finder.plist`). And It's easier to look some file's thumbnail rather than reading the title sometimes (for ex: an image directory). 
So, If you add these two feature, this would be awesome. Thank you again for your hard work.

**Comment by wtf403** [comment]
Yes, it would be great to have these features. But how will it install apps with an unloaded Finder. It looks like Installer.app doesn't work without Finder
